testing/types/expected_keepers.go

```diff
@@ -0,0 +1,22 @@
+package types
```

Comment:
I'm not so sure we need a separate directory for this. I think it makes more sense just to add to the top level `ibc-go/testing` directory

Reply:
left in order to avoid circular imports